subscription-manager.spec

```diff
@@ -79,15 +78,8 @@
 %global gtk3 1
 %endif
 
-%if 0%{?rhel} == 6
-%global use_initial_setup 0
-%global use_firstboot 1
-%global use_inotify 0
```

Comment:
As before: while it is good, I'd leave the removal of RHEL 6 bits to a different change.